### PR TITLE
Updating type search for getContent

### PR DESCRIPTION
**Pull request by johnhebron**
Playing around with `/library/sections/:sectionId` and in the returned Directory it shows the different type searches. I've updated the client to accept all four that I'm aware of.

- `search?type=1`: Search only (type=1) movies.
- `search?type=2`: Search only (type=2) shows.
- `search?type=3`: Search only (type=3) seasons.
- `search?type=4`: Search only (type=4) episodes.

Response Directory:
```
{
    "prompt": "Search for Shows",
    "search": true,
    "key": "search?type=2",
    "title": "Search Shows..."
},
{
    "prompt": "Search for Episodes",
    "search": true,
    "key": "search?type=4",
    "title": "Search Episodes..."
}
```

### CODE REVIEW

**Comment by LukeHagar** [comment]
Hey John!

Great catch! 

With this new data found search should probably be broken out into its own endpoint, and the type should be a query parameter. 

If you want to take a stab at it you can refer to the similar section/refresh endpoint.

Or I can take a crack at it later tonight.

Cheers 🎉

**Comment by johnhebron** [comment]
@LukeHagar I took a first stab and created `pms/paths/library-content-search.yaml`. 

I copied the query content from the `pms/paths/library-content.yaml` file since it seems they use all the same filters, which are all optional.

I'm still learning the API, so i'm not 100% on this stuff, so please 👀  for me, if you can.

Thank you!

File: pms/paths/library-content-search.yaml [hunk 1]
```diff
@@ -0,0 +1,201 @@
+get:
+  tags:
+    - Library
+  summary: Search Library Section Items
```
Comment (LukeHagar) [inline pms/paths/library-content-search.yaml:4]: Let's leave both the summary and the operation ID at just Search Library and searchLibrary

That should generate a more logical command verb structure.

Comment (johnhebron) [reply pms/paths/library-content-search.yaml]: @LukeHagar  👍 done an done

**Review by LukeHagar — APPROVED** [review]

**Pull request merged**